CHANGELOG.md

```diff
@@ -8,6 +8,19 @@ Future Todo List
 - Make typography and utility classes silent extenders (so that they can be extended by components without importing all utility classes).
 
 
+v6.0.0-beta.4
+------------------------------
+*September 23, 2021*
+
+### Changed
+- Button styles in line with icing phase 2.
+Now button modifiers are:
+- Btn Background Colour modifiers: `o-btn--primary`, `o-btn--secondary`, `o-btn--outline`, `o-btn--ghost`.
```

Comment:
It would be useful to list the specific things that have been removed so that anyone updating knows specifically about these breaking changes.

The main one I can see is `o-btn--tertiary` being removed, so it's worth calling that out under a removed sections here.

Similarly, I'd put the button size modifiers, `o-btn--outline` and `o-btn--ghost` under an `Added` section, just so it's clear what got added and removed in each release. This can be in addition to the Changed block you've already added, just makes it super clear what is new, and what is bein removed.